src/terraform/install.sh

```diff
@@ -153,7 +153,7 @@ if [ "${TERRAFORM_SHA256}" != "dev-mode" ]; then
     if [ "${TERRAFORM_SHA256}" = "automatic" ]; then
         receive_gpg_keys TERRAFORM_GPG_KEY
         curl -sSL -o terraform_SHA256SUMS https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_SHA256SUMS 
-        curl -sSL -o terraform_SHA256SUMS.sig https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_SHA256SUMS.${TERRAFORM_GPG_KEY}.sig
+        curl -sSL -o terraform_SHA256SUMS.sig https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_SHA256SUMS.sig
```

Comment:
ℹ️  It's more secure to always download the `.sig` including the `TERRAFORM_GPG_KEY`.

I understand the reason for changing that as https://releases.hashicorp.com/terraform/0.9.0/ doesn't include a .sig with the proposed key in this PR.

Reply:
I agree with you but there is also no file with the 64-bit version for the latest terraform version: https://releases.hashicorp.com/terraform/1.3.7/